.changeset/whole-baboons-do.md

```diff
@@ -0,0 +1,49 @@
+---
+"@comet/admin": minor
+---
+
+Add new `DateTimeRangePicker` and `DateTimeRangePickerField` components
+
+The new components are based on the `@mui/x-date-pickers-pro` package, so you can refer to the [MUI documentation](https://v7.mui.com/x/api/date-pickers/date-time-range-picker/) for more details.
+Unlike the MUI components, these components use an object with `start` and `end` properties, both of which use a `Date` object as the value, instead of an array of two `Date` objects.
```

Comment:
Is it a good idea to use a different format than MUI? Why did you decide to change it?